manim/_config/utils.py

```diff
@@ -692,6 +693,17 @@ def digest_args(self, args: argparse.Namespace) -> "ManimConfig":
             if hasattr(args, "media_dir") and args.media_dir:
                 self.media_dir = args.media_dir
 
+        # Handle --tex_template
+        if args.tex_template:
+            if args.tex_template == "ctex":
+                self.tex_template = TexTemplateLibrary.ctex
+            elif args.tex_template == "simple":
+                self.tex_template = TexTemplateLibrary.simple
+            elif args.tex_template == "threeb1b":
+                self.tex_template = TexTemplateLibrary.threeb1b
+            elif args.tex_template == "default":
+                self.tex_template = TexTemplateLibrary.default
+
```

Comment:
@ManimCommunity/latex could you please check if this is the preferred way of handling this flag?

Reply:
When first introduced, its aim was for the user to specify a custom template **file**, but I think with @cobordism's improvements, this is now the way to go. He knows better.